css/style.css

```
@@ -1509,19 +1751,22 @@ a {
 
 /* MODALS */
 .modal {
-  /* display: none; */
+  display: none;
   color: var(--black);
   background-color: var(--white);
   background-position: 0 0;
   background-repeat: repeat;
-  box-shadow: 0 10px 10px var(--shadow-black);
-  position: relative;
+  box-shadow: 0 5px 12px var(--shadow-black);
+  position: absolute;
```

```suggestion
  position: fixed;
```

попап должен быть по центру относительно экрана

---

Исправлено